runtime/doc/tips.txt

```diff
@@ -406,20 +405,6 @@ when they are not in the same location as the compressed "doc" directory.  See
 |$VIMRUNTIME|.
 
 ==============================================================================
-Executing shell commands in a window			*shell-window*
-
-There have been questions for the possibility to execute a shell in a window
-inside Vim.  The answer: you can't!  Including this would add a lot of code to
-Vim, which is a good reason not to do this.  After all, Vim is an editor, it
-is not supposed to do non-editing tasks.  However, to get something like this,
-you might try splitting your terminal screen or display window with the
-"splitvt" program.  You can probably find it on some ftp server.  The person
-that knows more about this is Sam Lantinga <slouken@cs.ucdavis.edu>.
-An alternative is the "window" command, found on BSD Unix systems, which
-supports multiple overlapped windows.  Or the "screen" program, found at
-www.uni-erlangen.de, which supports a stack of windows.
```

Comment:
> I just mean: we added a terminal. Why can't we say we added it?

Well we do in the nvim docs, but I see your point. I still think it's a little smug because of `... The answer: you can't! ...`, but I'll add it.